tests/test_convention.py

```diff
@@ -160,7 +160,8 @@ def test_cache():
             keypoints=coco_wb_keypoints2d,
             mask=coco_wb_mask,
             src='coco_wholebody',
-            dst=dst_key)
+            dst=dst_key,
+            read_cache=False)
```

Comment:
We may need to really figure out why using cache is occasionally slower than building mapping from scratch. Otherwise I'm afraid the same test assertion error will still occur.

Reply:
I'd like to treat it as feature of github machines. It also occurred to test_ffmpeg before, which was fixed before release. See commit below:
https://github.com/open-mmlab/mmhuman3d/commit/7d31a8f6dcf1b70e5fdb1bf5b466cb555a849ba4#diff-36a4195b22e2684e56bc3cb1924d34e7b496b92c83228d638f85e2a1dafc5efbL361
When we did tests on gitlab-CI, the time comparison never failed.

Reply:
It that's the case, should we remove the time comparison?

Reply:
Good idea. Removal of time comparison will not reduce the coverage. Do you approve it?

Reply:
I think we have no better idea, let's involve @yl-1993 in the discussion.

Reply:
I agree. I think we can change assert to warning and still keep the test. It will not influence the coverage.